src/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Loaderboard websit</title>
+</head>
+
+<body>
+    <div class="main">
+        <header>
+            <nav>
+                <input type="search" name="search" id="search" placeholder="search">
+                <input type="url" name="" id="" placeholder="http://">
+                <img src="images/up" alt="">
+                <img src="images/back" alt="">
+                <img src="images/close" alt="">
+                <img src="images/front" alt="">
+            </nav>
+        </header>
+        <section>
+            <h1>Loaderboard</h1>
```

Comment:
- [OPTIONAL] Kindly double-check for typos in your code before submitting the PR. 👍